sunpycube/cube/datacube.py

```diff
@@ -639,3 +640,13 @@ def plot_wavelength_slice(self, offset, **kwargs):
             offset, cumul_cube_lengths)
         plot = self[sequence_index].plot_wavelength_slice(cube_index, **kwargs)
         return plot
+
+
+class Slice_helper(object):
+    """Helper class to make slicing in index_as_cube more pythonic"""
+
+    def __init__(self, seq):
+        self.seq = seq
```

Comment:
Can you also explain in a comment or docstring what ```seq``` is?  Just a line should do.